pkgs/by-name/na/nawk/package.nix

```diff
@@ -1,28 +1,19 @@
-{ lib
-, stdenv
-, fetchFromGitHub
-, bison
-, buildPackages
-, installShellFiles
-}:
+{ lib, stdenv, fetchFromGitHub, bison, buildPackages, installShellFiles }:
```

Comment:
1. What is happening with the Nix formatter? Lists like these should be in a form similar to

```nix
{
  lib,
  stdenv,
  fetchFromGitHub,
  . . .
}
```

Reply:
🤦 I was running `nix run nixpkgs#nixfmt` thinking this was already using the new RFC style…